src/index.js

```diff
@@ -18,9 +18,11 @@ const sortDownIcon = <FontAwesomeIcon icon={faSortDown} />
 const App = () => {
     const [todos, setToDos] = useState([]);
     const [sortConfig, setSortConfig] = useState({});
-
-    const toDoInput = useRef();
-    const dueDateInput = useRef();
+    const [todoInputValue, setTodoInputValue] = useState('');
+    const [dateInputValue, setDateInputValue] = useState('');
+    
+    const isAddButtonDisabled = !todoInputValue || !dateInputValue;
+    const isClearButtonDisabled = !todos.some(({ completed }) => completed);
```

Comment:
Now I'm reading it again. you can also name this variable as `hasCompletedItem` something like that and later on:
```js
<button onClick={handleClearItems} disabled={hasCompletedItem}> Clear Selected Items </button>
```
For me variable name should reflect what information is stored inside, but of course we shouldn't bring it to the extreme. Always just name what you think will help you understand the most 3 months later lol.
There's no right or wrong here. What you have now is already really good.